src/util.rs

```diff
@@ -66,13 +67,13 @@ pub fn exec_cmd(cmd: &str, cmd_args: &[&str]) -> Result<String, String> {
 	}
 }
 
-/**
- * Sets up the panic hook for the terminal.
- *
- * See <https://ratatui.rs/how-to/develop-apps/panic-hooks/#termion>
- *
- * @return Result
- */
+
+/// Sets up the panic hook for the terminal.
+
+/// See <https://ratatui.rs/how-to/develop-apps/panic-hooks/#termion>
+ 
+/// @return Result
```

Comment:
Ah, I think we can also get rid of these C-style `@return` and `@param` lines too. Can you remove them as well?

Reply:
Do you mean completely removing those lines , if am not wrong

Reply:
Yes, we can completely remove them.

Reply:
> Ah, I think we can also get rid of these C-style `@return` and `@param` lines too. Can you remove them as well?

please review and let me know .